### DISCUSSION TITLE
block

### DISCUSSION THREAD

**Discussion by acqwua** [3kh0/ext-remover#783] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
nahh bro im done for
my school blocked EVERYTHING including chrome:// links

**Comment by AshtonDavies** [comment, marked as answer]
> nahh bro im done for
> my school blocked EVERYTHING including chrome:// links

Dude, you have plenty of solutions right at your fingertips and complain that it's over without trying. The readme actively gets updated with the help of me and @Brandon421-ops discovering and collecting just so you can pretend there's no way around patches.
